game/static/game/css/landing.css

```
@@ -73,345 +101,938 @@ body {
+    opacity: 0.04;
+    animation: floatPiece 12s ease-in-out infinite;
+}
+
+.fp-1 { top: 10%; left: 8%; animation-delay: 0s; font-size: 3rem; }
+.fp-2 { top: 20%; right: 10%; animation-delay: -2s; font-size: 2.8rem; }
+.fp-3 { bottom: 25%; left: 12%; animation-delay: -4s; }
+.fp-4 { bottom: 15%; right: 8%; animation-delay: -6s; font-size: 2.2rem; }
+.fp-5 { top: 50%; left: 3%; animation-delay: -8s; font-size: 2rem; }
+.fp-6 { top: 40%; right: 5%; animation-delay: -10s; font-size: 1.8rem; }
+
+@keyframes floatPiece {
```

Stylelint (17.11.0)

[error] Expected keyframe name "floatPiece" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)